### ISSUE TITLE
scalelite-recording-importer don't process some files

### ISSUE THREAD

**Issue by dseral** [blindsidenetworks/scalelite#165]
Today I finished and setup scalelite environment and i was tried to transfer my old published  (this BBB server only installed  3 week ago) recording to scalelite with scalelite_batch_import.sh command.
This script works fine but later on while i tried to check my spool directory i saw many tar archive files that not processed. So while i was investigate i saw following errors in docker log;
I, [2020-04-19T21:08:40.899696 #6]  INFO -- : Found metadata file: presentation/9d4fb9ed5ef0da1972fe011396495fa358683bd0-1584973297534/metadata.xml
E, [2020-04-19T21:08:40.922059 #6] ERROR -- : Failed to import recording: bad URI(is not URI?): "\n      https://xxx.xxx.xxx.xxx/playback/presentation/2.0/playback.html?meetingId=9d4fb9ed5ef0da1972fe011396495fa358683bd0-1584973297534\n    "
 
(xxx.xxx.xxx.xxx side replaced by me) 
So while i checked metadata.xml file content i saw that:
 "\<link\>
      https://xxx.xxx.xxx.xxx/playback/presentation/2.0/playback.html?meetingId=011b9a713f11631bce5d3877d69a41458236a935-1586858571855
    \</link\>"

So there are some error in importer that not support some kind of metadata files. 

How can i fix this? 
Devrim

**Comment by SebastianZiegler** [comment]
Had the same issue
https://github.com/blindsidenetworks/scalelite/issues/96
I manually fixed the metadata.xml (cut out the \n) and started again.

I think this happend, if the recordings description are edited through moodle because this not happend to all of our recordings.

**Comment by relecand** [comment]
Same here:

If the metadata.xml has such an entry:

```
    <link>
      https://xxxxx.de/playback/presentation/2.0/playback.html?meetingId=..
    </link>
```

it failes to import. 

Should be like this

```
<link>https://xxxxx.de/playback/presentation/2.0/playback.html?meetingId=..</link>
```

**Comment by TheJJ** [comment]
Fixed with #163

**Comment by relecand** [comment]
One problem more in the metadata.xml:

```
     <image 
          alt="abc" height="136" width="176">
            https://xxxx.de/presentation/xxxxxxe0caef231558fcd5b964593-1584355180752/presentation/xxxxxxxbf6c37b6a4b5ae187f-1584355180780/thumbnails/thumb-1.png
</image>
          <image 
          alt="abc" height="136" width="176">
            https://xxxx.de/presentation/xxxxxxe0caef231558fcd5b964593-1584355180752/presentation/xxxxxx6c37b6a4b5ae187f-1584355180780/thumbnails/thumb-2.png
</image>
```

Error message

```
Apr 21 20:49:52 scalelite.xxxxx.de docker[24090]: E, [2020-04-21T18:49:52.173188 #6] ERROR -- : Failed to import recording: bad URI(is not URI?): "\n            https://xxxxx.de/presentation/xxxxxe9781624dfef533b5a3a-1585635267736/presentation/xxxxx10bf6c37b6a4b5ae187f-1585635267777/thumbnails/thumb-1.png\n"
```

**Comment by farhatahmad** [comment]
Fixed by #163
